iis-wf/iis-wf-citationmatching-direct/src/main/java/eu/dnetlib/iis/wf/citationmatching/direct/service/CitationMatchingDirectReporter.java

```diff
@@ -0,0 +1,64 @@
+package eu.dnetlib.iis.wf.citationmatching.direct.service;
+
+import org.apache.spark.api.java.JavaRDD;
+import org.apache.spark.api.java.JavaSparkContext;
+
+import com.google.common.collect.Lists;
+
+import eu.dnetlib.iis.common.citations.schemas.Citation;
+import eu.dnetlib.iis.common.schemas.ReportParam;
+import pl.edu.icm.sparkutils.avro.SparkAvroSaver;
+
+/**
+ * Reporter of citation matching direct job counters.<br/>
+ * It calculates citation matching direct counters and saves them
+ * as {@link ReportParam} datastore.
+ * 
+ * @author madryk
+ */
+public class CitationMatchingDirectReporter {
```

Comment:
I would add `Counter` phrase in this name to make it straight-forward in terms of functionality: `CitationMatchingDirectCounterReporter`. This may seem to be too wordy though... Since it is already in `*.citationmatching.direct.*` package we could make `CounterReporter` out of it. What do you think?